### PR TITLE
Win

### PR DESCRIPTION
**Pull request by stotiks**

### CODE REVIEW

File: include/vnx/addons/TcpServer.h [hunk 1]
```diff
@@ -95,7 +100,7 @@ class TcpServer : public TcpServerBase {
 private:
 	int m_socket = -1;
 #ifdef _WIN32
-	int m_notify_socket = -1;
+	SOCKET m_notify_socket = INVALID_SOCKET;
```
Comment (madMAx43v3r) [inline include/vnx/addons/TcpServer.h:103]: why not the same for `HttpServer.h` ?

Comment (stotiks) [reply include/vnx/addons/TcpServer.h]: Looks like this change not needed, reverted

**Comment by madMAx43v3r** [comment]
can close this right? it's all in master already.

**Comment by stotiks** [comment]
Yes

**Pull request closed without merging**